feature/experimental/backup_nhg/ate_tests/backup_nhg_test/README.md

```diff
@@ -9,28 +9,31 @@ containing a single NH.
 
 *   Connect ATE port-1 to DUT port-1, ATE port-2 to DUT port-2, and ATE port-3
     to DUT port-3.
-*   TODO: Create a non-default VRF, VRF-3, which includes DUT port-3.
-*   TODO: Create a static default route in the VRF-3, pointing to ATE port-3.
+*   Create a non-default VRF, BACKUP-NHG-1.  Assign DUT port-1 to BACKUP-NHG-1.
+*   All GRIBI NHGs and NHs will be installed to the default VRF.
 *   Connect gRIBI client to DUT with persistence `PRESERVE`, redundancy
     `SINGLE_PRIMARY`, with election ID 1.
-*   Install an IPv4Entry in the default VRF pointing to ATE port-2 for prefix
-    198.51.100.0/24, with a backup nexthop group pointing to ATE port-3 (TODO:
-    change ATE port-3 to VRF-3).
+
+To validate GRIBI backup next hop group behavior:
+
+*   Install a GRIBI IPv4Entry in vrf BACKUP-NHG-1 for the prefix 198.51.100.0/24 pointing to ATE port-2, with the backup nexthop group pointing to the default VRF.
+*   Install a GRIBI IPv4Entry in the default VRF for the prefix 198.51.100.0/24 pointing to ATE port-3.
 *   Validate:
     *   AFT telemetry shows next-hop-group of DUT port-2 being selected for
         198.51.100.0/24.
     *   Traffic is forwarded to ATE port-2 from ATE port-1.
-*   For each of the following cases, ensure that traffic switches to being
-    forwarded to ATE port-3:
+*   For each of the following cases, ensure that traffic switches to using ATE port-3:
     *   Interface ATE port-2 is disabled.
     *   Interface DUT port-2 is disabled.
-*   Remove all previously installed IPv4Entry routes. Create an entry for
-    198.51.100.0/24 with a next-hop of 192.0.2.254/32. Inject a second entry
-    with 192.0.2.254/32 resolved to ATE port-2. Specify a backup NHG pointing to
-    ATE port-3 (TODO: change ATE port-3 to VRF-3) for the 198.51.100.0/24
-    entry’s NHG.
-    *   Remove the entry for 192.0.2.254/32, and ensure that traffic is
-        forwarded to ATE port-3 for destinations in 198.51.100.0/24.
+
+To validate hierarchical backup next hop group behavior:
```

Comment:
let's investigate the default 0/0 solution/impact.

Reply:
I think the solution we discussed will work, so I removed the deviation/skip feature for the hierarchical test.